src/bitcoin-cli.cpp

```diff
@@ -571,6 +574,14 @@ class NetinfoRequestHandler : public BaseRequestHandler
                 reachable_networks.push_back(network_id);
             }
         };
+
+        for (const size_t network_id : UNREACHABLE_NETWORK_IDS) {
+            if (m_counts.at(2).at(network_id)) {
+                result += strprintf("%8s", NETWORK_SHORT_NAMES.at(network_id)); // column header
+                reachable_networks.push_back(network_id);
+            }
+        }
```

Comment:
Can ignore, the following version might be more idiomatic C++.

```diff
-        for (const size_t network_id : UNREACHABLE_NETWORK_IDS) {
-            if (m_counts.at(2).at(network_id)) {
-                result += strprintf("%8s", NETWORK_SHORT_NAMES.at(network_id)); // column header
-                reachable_networks.push_back(network_id);
-            }
+        for (size_t network_id : UNREACHABLE_NETWORK_IDS) {
+            if (m_counts.at(2).at(network_id) == 0) continue;
+            result += strprintf("%8s", NETWORK_SHORT_NAMES.at(network_id)); // column header
+            reachable_networks.push_back(network_id);
         }
```

Reply:
Thanks! both suggestions applied.